src/wasm/kotlin/javatests/arcs/UnicodeTest.kt

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2019 Google LLC.
+ *
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ *
+ * Code distributed by Google as part of this project is also subject to an additional IP rights
+ * grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+package sdk.kotlin.javatests.arcs
+
+import arcs.addressable.toAddress
+import arcs.Collection
+import arcs.Handle
+import arcs.Particle
+import arcs.Singleton
+import kotlin.native.internal.ExportForCppRuntime
+import kotlin.native.Retain
+
+class UnicodeTest : Particle() {
```

Comment:
This seems to be a dependency of the actual test, in that case - it's generally preferred to place it within the `java` directory, rather than `javatests`.  I think the guidance I've heard is "nothing should ever depend on something within `javatests`".

Reply:
That practice is documented here: http://go/suzhang-readability-snippets#shared-utility-file-location

Reply:
Hmm, not sure that's really the right fit here. These are the files under test, and aren't themselves being shared by multiple BUILD packages. The separation of these source files and the test "driver" wasm-api-test.ts is due to both our somewhat unusual node.js testing framework, and the need to test multiple language-specific wasm modules with the same driver file.

IMO moving them to the real source tree is obfuscating.

Reply:
Ok, I guess it's fine for this one... but if we ever end up using it in a `deps` declaration within a build target - we should consider moving it to the `java` dir (under a `something.whatever.testutils` package)